.mvn/extensions.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<extensions>
+    <extension>
+        <groupId>org.jboss.fuse.maven</groupId>
+        <artifactId>maven-exclude-extension</artifactId>
+        <version>1.0-SNAPSHOT</version>
```

Comment:
Where is this snapshot supposed to come from?

Reply:
This is temporary - I've added the build of maven-exclude-extension to the pipeline, so this will be replaced in-build with the pipeline build of maven-exclude-extension -

https://github.com/jboss-fuse/fuse-jenkins-pipelines/blob/main/8.5/projects-common.json#L2-L15